src/Popover.tsx

```diff
@@ -325,7 +325,7 @@ class JSModalPopover extends Component<JSModalPopoverProps, ModalPopoverState> {
       return (
         <View
           pointerEvents="box-none"
-          style={[styles.container, { left: 0 }]}
```

Comment:
{ left: 0 } is not needed as styles.container already specifies a 0px position for left.